pkgdown/_pkgdown.yml

```diff
@@ -25,6 +33,7 @@ reference:
   - dm_get_all_pks
   - dm_rm_pk
   - dm_enum_pk_candidates
+  - enum_pk_candidates
```

Comment:
Can you please collect all non-dm functions in a separate sub-topic?

Reply:
The `enum_(p,f)k_candidates()` functions work for `zoomed_dm` as well as for `tibble/data.frame`, just saying :)

So generally functions that don't immediately have anything to do with a `dm` should get their own sub-topic? We already have some functions like that (`check_cardinality()`, `check_key()` etc.)

Reply:
Does the `dm_` prefix mean that the first argument is a dm (or a zoomed dm)?

Reply:
It should IMHO mean, that the first argument is a `dm`, and  (almost always) not a `zoomed_dm`. The latter should normally be treated like a `tibble`.

Having said that, so far it is e.g. possible to set keys for a `zoomed_dm` (independent of which table is zoomed) and probably a few other functions will work too.

If we wanted to be really strict, we could say, that we don't allow anything on a `zoomed_dm` that works on a `dm`. That way it would be easier to be consistent with the prefixes.(?)

Reply:
Agree to be stricter: functions/methods that apply to a zoomed dm don't have a prefix; functions work either for a zoomed, or for a non-zoomed dm.

Reply:
outsourced the point "Can you please collect all non-dm functions in a separate sub-topic?" into issue #231.

outsourced "Agree to be stricter: functions/methods that apply to a zoomed dm don't have a prefix; functions work either for a zoomed, or for a non-zoomed dm." into issue #232